button/lib/button.ts

```diff
@@ -29,6 +29,10 @@ export abstract class Button extends LitElement implements ButtonState {
   static override shadowRootOptions:
       ShadowRootInit = {mode: 'open', delegatesFocus: true};
 
+  @property({type: String, attribute: 'data-aria-expanded', noAccessor: true})
+  @ariaProperty
+  override ariaExpanded!: ARIAExpanded;
+
   @property({type: String, attribute: 'data-aria-has-popup', noAccessor: true})
```

Comment:
Unrelated, but this seems like an error to me; `haspopup` should not be hyphenated.